ckanext/scheming/templates/scheming/form_snippets/datetime.html

```diff
@@ -0,0 +1,42 @@
+{% import 'macros/form.html' as form %}
+{% set date = '' %}
+{% set time = '' %}
+{% set dateTime = data.get(field.field_name, '') %}
+
+{% if dateTime is string %}
+    {% set parts = dateTime.split() %}
+    {% set date = parts[0] %}
+    {% set time = parts[1] %}
+{% elif dateTime is sequence %}
+    {% set date = dateTime[0] %}
+    {% set time = dateTime[1] %}
+{% endif %}
+{% call form.input(
+    field.field_name,
+    id='field-' + field.field_name +  '-date',
+    label= h.scheming_language_text(field.label),
+    type='date',
+    value=date,
+    error=errors[field.field_name],
+    classes=['control-medium'],
+    attrs=field.form_attrs if 'form_attrs' in field else {},
+    is_required=h.scheming_field_required(field)
+    )
+%}
+    {%- snippet 'scheming/form_snippets/help_text.html', field=field -%}
+{% endcall %}
+
+{% call form.input(
+    field.field_name,
+    id='field-' + field.field_name + '-time',
+    label= h.scheming_language_text(field.get('label_time', 'Time')),
+    type='time',
+    value=time,
+    error=errors[field.field_name],
```

Comment:
Actually, it's very bad if the error is on only one field because of the error is in the time field and that field is not identified as having an error, the user will have a hard time figuring out that the issue is with a field not highlighted. Ideally we should narrow the error to one of the other field but I haven't quite figured out how to do that yet.